zcash_test_vectors/orchard_zsa/key_components.py

```diff
@@ -46,50 +47,42 @@ def randbytes(l):
 
     test_vectors = []
     for i in range(0, 10):
-        sk = SpendingKey(rand.b(32))
+        ki = KeyInit(rand)
```

Comment:
`ki` sounds like a typo of `key`, let's do `k` instead.

Reply:
Done.